### ISSUE TITLE
GCop139 false alarm

### ISSUE THREAD

**Issue by sungam3r** [Geeksltd/GCop#260]
```csharp
internal class PlatformHelper
{
    private static readonly string[] _platforms = new[] { "netstandard2.0", "net462", "netcoreapp2.0", "netcoreapp2.1" }; // GCop139	Use constant instead of field.		
}
```

But it is obvious that the array field cannot be constant.

**Comment by watsonsong** [comment]
I met the same problem, it seem always alarm when met native array type.

**Comment by sungam3r** [comment]
One more example:
```c#
internal struct ValueStopwatch
{
        private static readonly double TimestampToTicks = TimeSpan.TicksPerSecond / (double)Stopwatch.Frequency; // GCop139
}

**Comment by sungam3r** [comment]
@Karvan Any progress on this?

**Comment by marjanjavid** [comment]
> array field

The first problem is fixed on development branch, please wait for the release.

**Comment by Karvan** [comment]
@azadimehdi could you please see if we still kick this : 

https://github.com/Geeksltd/GCop/issues/260#issuecomment-493580727

if it's been fixed we can close it.

**Comment by azadimehdi** [comment]
@Karvan there was a problem , but I've pushed some new changes , it's been fixed now.
